cpu/esp32/usb_reset.c

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2023 Benjamin Valentin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     cpu_esp32
+ * @{
+ *
+ * @file
+ * @brief       Trigger reset to the bootloader stored in the internal boot ROM
+ *              memory.
+ *
+ *              This will start the USB/UART bootloader.
+ *
+ * @author      Benjamin Valentin <benpicco@googlemail.com>
+ * @author      Gunar Schorcht <gunar@schorcht.net>
+ *
+ * @}
+ */
+
+#include "rom/usb/chip_usb_dw_wrapper.h"
+#include "rom/usb/usb_persist.h"
+#include "soc/rtc_cntl_reg.h"
+#include "periph/pm.h"
```

Comment:
```suggestion
#include "driver/periph_ctrl.h"
#include "rom/usb/chip_usb_dw_wrapper.h"
#include "rom/usb/usb_persist.h"
#include "soc/rtc_cntl_reg.h"
#include "soc/rtc_cntl_struct.h"
#include "soc/usb_serial_jtag_reg.h"
#include "soc/usb_serial_jtag_struct.h"

#include "periph/pm.h"
```
I guess these includes are required to be able to combile the code for ESP32-S3.